contracts/AelinPool.sol

```diff
@@ -399,6 +399,7 @@ contract AelinPool is AelinERC20, MinimalProxyFactory {
             require(acceptAmount <= maxOpen, "accepting more than share");
         }
         totalAmountAccepted += acceptAmount;
+        amountAccepted[recipient] += acceptAmount;
```

Comment:
Is it from an old commit?

Reply:
it is something that we need to add for the UI but doesn't affect the pool logic at all

Reply:
I can take it out and add it to the other open PR but it is small enough that we can tack it on to either open PR imo